Dockerfile

```diff
@@ -79,6 +79,11 @@ ENV _APP_SERVER=swoole \
     _APP_STORAGE_ANTIVIRUS=enabled \
     _APP_STORAGE_ANTIVIRUS_HOST=clamav \
     _APP_STORAGE_ANTIVIRUS_PORT=3310 \
+    _APP_STORAGE_DEVICE=local \
+    # _APP_STORAGE_DEVICE_S3_ACCESS_KEY=s3-access-key \
```

Comment:
I think you can uncomment this vars

Reply:
@eldadfux if we uncomment how to use empty default value there?